src/morebv.jl

```diff
@@ -41,7 +41,9 @@ function morebv(n :: Int=100)
     @NLobjective(
       nlp,
       Min,
-	    sum((2.0 * x[i] - x[i-1] - x[i+1] + (h^2 / 2.0) * (x[i] + (i - 1) * h + 1)^3)^2 for i=2:n-1)
+        sum((2.0 * x[i] - x[i-1] - x[i+1] + (h^2 / 2.0) * (x[i] + (i - 1) * h + 1)^3)^2 for i=2:n-1) +
```

Comment:
Thanks for catching this! Based on Buckley's report, I think this term should have `i * h` instead of `(i - 1) * h`, and the last end-point term should have `n * h` instead of `(n - 1) * h`. Also `h = 1 / (n+1)`. Lots of errors here!

![Screen Shot 2019-10-13 at 13 02 59](https://user-images.githubusercontent.com/38760/66719153-daf91380-edb9-11e9-90f4-4bac521e1793.png)

Reply:
Thanks for the further improvements. I now think the initialization setting `x0[1] = x0[n] = 0` was  a misreading of the problem, and that http://www.cs.cas.cz/matonoha/download/V1081.pdf suggests starting all points at 0.5. I made that further change. But note that https://github.com/mpf/Optimization-Test-Problems/blob/master/cute/morebv.mod seems to use a very different initialization.

Reply:
Ok. Let's add a note about the initial point that you choose, if you don't mind.